hawkbit-dmf/hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/integration/AmqpAuthenticationMessageHandlerIntegrationTest.java

```diff
@@ -45,8 +47,8 @@
 
 @Feature("Component Tests - Device Management Federation API")
 @Story("Amqp Authentication Message Handler")
-@SpringApplicationConfiguration(classes = { RepositoryApplicationConfiguration.class, AmqpTestConfiguration.class,
-        DmfApiConfiguration.class })
+@SpringBootTest(classes = { DmfApiConfiguration.class, RepositoryApplicationConfiguration.class,
```

Comment:
You do not need to declare TestConfiguration and TestSupportBinderAutoConfiguration here if you declare them with the ContextConfiguration annotation in the super class AbstractAmqpIntegrationTest.

Reply:
see above